composer.json

```diff
@@ -80,6 +80,7 @@
         "php-amqplib/php-amqplib": "^3.2",
         "phpseclib/mcrypt_compat": "^2.0",
         "phpseclib/phpseclib": "^3.0",
+        "phpstan/phpstan-deprecation-rules": "^1.1",
```

Comment:
Please move this to the `require-dev` section, just like the `phpstan/phpstan` dependency. This shouldn't be a production requirement, but a dev requirement 🙂

Reply:
Hi @hostep, I updated.